demo-app/src/components/aruba/ArubaData.js

```diff
@@ -0,0 +1,21 @@
+const type = ['Branch', 'Campus', 'Main', 'Data center'];
```

Comment:
I like having all of these dependent files in a single directory -- keeps the domain confined.

A couple suggestions:
- Let's genericize the branding by replacing "aruba" references with "networking"
- create a `data` sub directory even if its a single file
- reserve Title case file names for components

Reply:
Done